src/main/java/avd/inf/jdm/rentmycar/service/CarService.java

```diff
@@ -27,25 +27,25 @@ public CarService(CarRepository carRepository, UserRepository userRepository) {
         this.userRepository = userRepository;
     }
 
-    public List<Car> getAllCars() {
+    public List<Car> getAll() {
         return carRepository.findAll();
     }
 
-    public Car getCarById(Long id) {
+    public Car getById(Long id) {
         return carRepository.findById(id).get();
     }
 
     public Optional<Car> getSingleById(Long id) {
         return carRepository.findById(id);
     }
 
-    public Boolean existCarById(Long id){
+    public Boolean existsById(Long id){
```

Comment:
I think you could get the same result with `getById(id).isPresent()`, but this is fine as well 👍